### PR TITLE
Update UG for add, edit, delete commands

### PR DESCRIPTION
**Pull request by zi-yii**
The UG format for add, edit, delete commands have been updated to match the new format to make the UG clearer.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -79,16 +79,49 @@ Adds a client to the address book.
...
 
 <box type="tip" seamless>
+    Parameters
+</box>
+
+1. `INDEX` : Must be a valid client index in the filtered list that is currently shown.
+2. `NAME` : Must be alphanumeric, can include spaces.
+3. `PHONE_NUMBER` : Must be numeric and at least 3 digits long.
+4. `EMAIl` : Must ba a valid email address following the format `local-part@domain`.
+5. `ADDRESS`: Can take any values.
```
Comment (RezwanAhmed123) [inline docs/UserGuide.md:89]: maybe we can exclude characters like "/" though for address.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -79,16 +79,49 @@ Adds a client to the address book.
...
+<box type="warning" seamless>
+    <span circle slot="icon"><md>:bulb:</md></span>
+    Note the following:
+</box>
+
+* If the `INDEX` is invalid, the user will be informed with an error message.
+* Parameters `NAME`, `PHONE_NUMBER`, `EMAIl`and `ADDRESS` cannot be blank.
+* A client can have any number of tags (including 0).
+* Phone number can take on multiple formats, including Singapore and foreign
+  numbers (without the + symbol). This supports insurance agents with an international customer base.
```
Comment (RezwanAhmed123) [inline docs/UserGuide.md:102]: maybe to add onto this, we can put in the known issues section that the currency of claim may not always be singapore and that the possible remedy is to convert by yourself first before putting into the app?

**Review by RezwanAhmed123 — COMMENTED** [review]
LGTM, some nits

File: docs/UserGuide.md (hunk 2 above)
Comment (matthewyip1511) [reply docs/UserGuide.md:102]: Good point, but did you refer to the wrong part? This part is talking about phone numbers 😅

File: docs/UserGuide.md (hunk 1 above)
Comment (zi-yii) [reply docs/UserGuide.md:89]: Addresses can take "/"

File: docs/UserGuide.md (hunk 2 above)
Comment (zi-yii) [reply docs/UserGuide.md:102]: Sures!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ InSUREance is a **desktop app for managing clients, optimized for use via a Comm
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a message explaining how to access the help page.
```
Comment (matthewyip1511) [inline docs/UserGuide.md:68]: Great catch!

**Review by matthewyip1511 — APPROVED** [review]
LGTM, but will wait for @RezwanAhmed123 to take a look before merging 🙂

**Review by RezwanAhmed123 — APPROVED** [review]
LGTM

**Pull request merged**